File: 백준/Bronze/16199. 나이 계산하기/나이 계산하기.cc
#include <iostream>
using namespace std;
int y, m, d;
int ty, tm, td;

int main() {
    cin >> y >> m >> d;
    cin >> ty >> tm >> td;
    
    int birthday = (y*365) + (m*30) + d;
    int today = (ty*365) + (tm*30) + td;
    cout << (today-birthday)/365 << endl;
    cout << ty-y+1 << endl;
    cout << ty-y << endl;
}

File: 백준/Bronze/16199. 나이 계산하기/나이 계산하기.swift
import Foundation

let (y,m,d) = [readLine()!.split(separator: " ").map{Int($0)!}].map{($0[0], $0[1], $0[2])}[0]
let (ty,tm,td) = [readLine()!.split(separator: " ").map{Int($0)!}].map{($0[0], $0[1], $0[2])}[0]

let birthday = (y*365)+(m*30)+(d)
let today = (ty*365)+(tm*30)+(td)

print((today - birthday)/365)
print(ty-y+1)
print(ty-y)